web/src/engine/websites/decorators/Common.ts

```diff
@@ -521,6 +522,35 @@ export function ImageElement(includeRefererHeader = true, detectMimeType = false
     };
 }
 
+export async function FetchImageAjaxFromHTML(page: Page, priority: Priority, signal: AbortSignal, queryImage: string, detectMimeType = false): Promise<Blob> {
```

Comment:
I recommend to add `this: MangaScraper` as first parameter to get type safety support